package.json

```diff
@@ -22,7 +22,7 @@
     "generate:sitemap": "next-sitemap",
     "export": "next export",
     "start": "next start",
-    "check": "concurrently --group --timings npm:check:* # Run all the npm check:* scripts in parallel",
+    "check": "npm exec concurrently -y -- --group --timings npm:check:* # Run all the npm check:* scripts in parallel",
```

Comment:
What's the reason for this change? 
All the npm installed executables are installed in `.node_modules/.bin` by `npm ci` and are in PATH when these `npm run` scripts are run, so why do you add the `npm exec ... -y` here?

Reply:
If the documentation in https://github.com/cert-manager/website#website-development-tooling is wrong, we should fix that.